plugins/modules/docker_image.py

```diff
@@ -665,17 +665,45 @@ def load_image(self):
 
         :return: image dict
         '''
+        # Load image(s) from file
+        load_output = []
         try:
             self.log("Opening image %s" % self.load_path)
             with open(self.load_path, 'rb') as image_tar:
                 self.log("Loading image from %s" % self.load_path)
-                self.client.load_image(image_tar)
+                for line in self.client.load_image(image_tar):
+                    self.log(line, pretty_print=True)
+                    if "stream" in line or "status" in line:
+                        load_line = line.get("stream") or line.get("status") or ''
+                        load_output.append(load_line)
```

Comment:
nitpickers of the world, unite!

I did not dig deeply to see how ``self.client.load_image()`` or whatever ``line`` is, but it does seem that the code in the PR is testing for ``stream`` and ``status`` twice, once explicitly in the ``if`` and once more implicitly in the ``get()`` (assuming a behaviour similar to ``dict``). If that's indeed the case, it might get clearer as:
```python
                    load_line = line.get("stream") or line.get("status")
                    if load_line:
                        load_output.append(load_line)
```

Reply:
If that's not the case, at the very least we should not need the ``or ''`` at the end of line 677 because we have tested for ``stream`` or ``status`` before, so we should never get to that last leg of the ``or``.

Reply:
Talking about nitpicking: `"status" in line` does not guarantee that `line.get("status")` is not falsish, or even `None` :)

The way the code is currently written, it will produce one string for every line that has `status` or `stream` in it, and it is identical to similar code further up (or down?) for building images. (Eventually I want to refactor it, but not yet :) )